.github/workflows/build-wheels-platforms.yml

```diff
@@ -0,0 +1,179 @@
+name: platforms-dispatches
+
+on:
+  schedule:
+    - cron: '0 0 * * 0,3'
+  workflow_dispatch:
+
+env:
+  MIN_IDF_MAJOR_VERSION: ${{ vars.MIN_IDF_MAJOR_VERSION }}
+  MIN_IDF_MINOR_VERSION: ${{ vars.MIN_IDF_MINOR_VERSION }}
+
+jobs:
+  build-python-wheels:
+    name: Build Python Wheels for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - windows-latest
+          - ubuntu-latest
+          - macos-latest
+          - linux-armv7-self-hosted
+          - macos-latest-xlarge       # MacOS M1 GitHub beta runner - paid $0.16
+          - BrnoRPI-GH004             # linux ARM64
+        include:
+          - os: linux-armv7-self-hosted
+            CONTAINER: python:3.8-bullseye
+          - os: BrnoRPI-GH004
```

Comment:
Could this be something like `linux-rpi-self-hosted`? We try to hide our server locations from our repos.

Reply:
Yes, sure. 
But I think we will need to add new tag for this runner, is it right @tomassebestik? Because `ARM64` tag is used for Linux and MacOS runners, the Linux machine is higher in the list but I am not sure which mechanism is used to select the correct runner.

Reply:
@jakub-kocka Normally we would use combination `self-hosted`+`arm64`, like for example [here](https://github.com/espressif/pytest-embedded/blob/main/.github/workflows/test-python.yml#L14-L16).

I'm not sure this would be easy in your workflow .... I have added new tag `linux-arm64-self-hosted` to BrnoRPI-GH004 runner, you can use that.

<img src="https://github.com/espressif/idf-python-wheels/assets/33937168/0aef1dbd-2d9b-4778-8ffa-d53ca4b5a184" width=500px>

Reply:
Thank you.